osu.Game/OsuGame.cs

```diff
@@ -85,7 +85,7 @@ private Intro intro
         private OnScreenDisplay onscreenDisplay;
 
         private Bindable<int> configRuleset;
-        public Bindable<RulesetInfo> Ruleset = new Bindable<RulesetInfo>();
+        private readonly Bindable<RulesetInfo> ruleset = new Bindable<RulesetInfo>();
```

Comment:
For now, I'd really like if we also exposed an `IBindable<RulesetInfo>` as a dependency, and then use that one wherever the ruleset isn't actually modified.

This way I think it'll ease our pain when we later remove the `Bindable<Ruleset>` from dependencies.